packages/fast-components-styles-msft/src/carousel/index.ts

```diff
@@ -82,62 +88,62 @@ const styles: ComponentStyles<CarouselClassNameContract, DesignSystem> = (
             "& $carousel_flipperPrevious, & $carousel_flipperNext": {
                 color: "black",
                 fill: "black",
-                background: "white",
+                background: "rgba(255, 255, 255, 0.6)",
+                border: "1px solid #CECECE",
                 "& span::before": {
                     borderColor: "black",
                 },
                 "&:hover": {
+                    background: "white",
                     "& span::before": {
                         borderColor: hoverContrast(config.contrast, "black"),
                     },
                 },
             },
             "& $carousel_sequenceIndicator": {
                 "&::before": {
-                    border: "1px solid white",
+                    background: "white",
+                    borderColor: "#CECECE",
                 },
-                "&:focus": {
+                "&$carousel_sequenceIndicator__active": {
                     "&::before": {
-                        boxShadow: "0 0 0 1px white",
+                        opacity: "1",
+                        background: "white",
                     },
                 },
             },
-            "& $carousel_sequenceIndicator__active": {
-                "&::before": {
-                    background: "white",
-                },
-            },
         },
         carousel__themeLight: {
             "& $carousel_flipperPrevious, & $carousel_flipperNext": {
                 color: "white",
                 fill: "white",
-                background: "black",
+                background: "rgba(0, 0, 0, 0.6)",
+                border: "1px solid #CECECE",
```

Comment:
@bheston can you advise if this is part of a larger change to flipper? If it is - @nalogiudice can we do the flipper change in a new PR and we can get this in and updated?

Reply:
We would still have to override here as regular flipper uses foreground and background colors, and this uses hard-coded color values

Reply:
Realizing that we still need to take in all the recent color work, what aspect of this requires the hard-coded colors? Is this to do with the light and dark theme support only? Perhaps that's something we can fix when we take in the new color work, or perhaps that indicates that we actually want the light/dark support in Flipper itself. I think any use we can think of for Flipper would benefit from the opacity, like for instance in something like PDF or other document navigation.
Also, if we do leave them hard coded here, the border should be 60% as well, so we should probably switch that to opacity on the backplate as a whole.

Reply:
Thanks both - sounds like we need to just tackle flipper in a new task unrelated to this? I assume the carousel work will get validated more when we do a pass with the new color stuff. I think the only action is ensuring that @bheston validates the final design changes for carousel specifically.

Reply:
> ... what aspect of this requires the hard-coded colors? Is this to do with the light and dark theme support only?

Correct. 

> or perhaps that indicates that we actually want the light/dark support in Flipper itself

Interesting thought - could be useful.

> I think the only action is ensuring that @bheston validates the final design changes for carousel specifically. - @chrisdholt 

Ok, so for this PR I will just validate the overrides are correct. When we update flipper, we can circle back and change what we need to here. Sound good?

Reply:
Good

Reply:
Works for me.

Reply:
@bheston `rgba(206, 206, 206, 0.6)` for the `#CECECE` border change?

Reply:
Looks about right to me with the rgba. Went ahead and made the change.